src/content/docs/data-apis/custom-data/intro-custom-data.mdx

```diff
@@ -9,12 +9,12 @@ To get the most out of New Relic, you may need or want to report custom data to
+Options for reporting custom data include: 
 
-* Use our [open source telemetry integrations](/docs/integrations/open-source-telemetry-integrations/get-started/introduction-new-relics-open-source-telemetry-integrations). We have integrations for reporting data from OpenTelemetry, DropWizard, Prometheus, and more. 
+* Our integrations for third-party telemetry services and specifications, including [OpenTelemetry](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic), [Prometheus](/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic), [Grafana](/docs/more-integrations/grafana-integrations/get-started/grafana-support-prometheus-promql), [DropWizard](/docs/more-integrations/open-source-telemetry-integrations/dropwizard/dropwizard-reporter), [Micrometer](/docs/more-integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry). [Browse all our solutions.](https://newrelic.com/instant-observability)
 * Configure our agents to send custom data. If you're using one of our APM agents, browser agents, or mobile agents, you can configure them to send custom data. To learn more, see the docs for the tool you're using. 
 * Create your own solution for reporting telemetry data with our language-specific [Telemetry SDKs](/docs/telemetry-data-platform/ingest-manage-data/ingest-apis/telemetry-sdks-report-custom-telemetry-data). Or use the underlying APIs directly: [Metric API](/docs/telemetry-data-platform/ingest-manage-data/ingest-apis/introduction-metric-api), [Trace API](/docs/distributed-tracing/trace-api/introduction-trace-api), [Event API](/docs/telemetry-data-platform/ingest-manage-data/ingest-apis/introduction-event-api), [Log API](/docs/logs/log-management/log-api/introduction-log-api). 
 * Report data from your operating system or network. If you're using our infrastructure agent, you might want to use our [Flex integration](/docs/integrations/host-integrations/host-integrations-list/flex-integration-tool-build-your-own-integration), which lets you use simple config files to report data.  
-* Build a [New Relic One application](/docs/new-relic-one/use-new-relic-one/build-new-relic-one/new-relic-one-build-your-own-custom-new-relic-one-application). You can build an app that lives on our platform, that can be shared with your team or the public, and that uses your own JavaScript UI functionality. You can analyze New Relic data, or bring in whatever data you want. 
+* Build a [New Relic One application](/docs/new-relic-one/use-new-relic-one/build-new-relic-one/new-relic-one-build-your-own-custom-new-relic-one-application) or customize an existing one from [the app catalog](https://opensource.newrelic.com/nerdpacks). You can build an app that lives on our platform, that can be shared with your team or the public, and that uses your own JavaScript UI functionality. You can analyze New Relic data, or bring in whatever data you want. 
```

Comment:
@zuluecho9, this final item in the list is really about a custom program with New Relic--not really about bringing in data like an integration, right? I wonder if this should be in the list here.

Reply:
they actually can use NR1 apps to bring in any sort of data from any arbitrary source (e.g., it's like building any website/app where you can bring in any data you want) so I consider it a type of solution for bringing in any custom data.